packages/stylist-core/src/ast.rs

```diff
@@ -49,18 +57,10 @@ impl ToCss for Scopes {
     }
 }
 
-impl FromStr for Scopes {
-    type Err = Error;
-
-    fn from_str(s: &str) -> Result<Self> {
-        Parser::parse(s)
-    }
-}
-
-#[derive(Debug, Clone, PartialEq)]
-pub(crate) struct Scope {
-    pub(crate) condition: Option<String>,
-    pub(crate) stylesets: Vec<ScopeContent>,
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Scope {
+    pub condition: Option<String>,
+    pub stylesets: Vec<ScopeContent>,
 }
```

Comment:
I'm still deliberating with myself if the fields should be public, or if we'd just want to expose public constructors (e.g. one with condition and one without) and mutable access for the stylesets part.